--- cloud-gate.spec	2020-05-05 15:10:40.000000000 -0700
+++ cloud-gate.spec.docker	2020-05-05 16:18:58.000000000 -0700
@@ -30,7 +30,7 @@
 
 %install
 #%make_install
-%{__install} -Dp -m0755 ~/go/bin/cloud-gate %{buildroot}%{_bindir}/cloud-gate
+%{__install} -Dp -m0755 /go/bin/cloud-gate %{buildroot}%{_bindir}/cloud-gate
 install -d %{buildroot}/usr/lib/systemd/system
 install -p -m 0644 misc/startup/cloud-gate.service %{buildroot}/usr/lib/systemd/system/cloud-gate.service
 install -d %{buildroot}/%{_datarootdir}/cloud-gate/customization_data/templates
